### PR TITLE
[HOTFIX]: Add manual trigger for cache updater stack

### PR DESCRIPTION
**Pull request by branberry**
### Notes
This PR resolves the issue related to `10gen` repositories not being built. For confirmation, I tested the process manually, and the updated cache files for `10gen` repos are now present. To verify, view the `snooty-parse-cache` S3 bucket in the AWS console, and confirm that the `.snooty-cloud-docs-...` ZIP file is present within the repository, and that it was updated today (February 23rd). 
### README updates

- - [x] This PR introduces changes that should be reflected in the README, and I have made those updates.

### CODE REVIEW

File: .github/workflows/deploy-prd-enhanced-cache.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 on:
   release:
     types: [released]
+  workflow_dispatch:
+    inputs:
+      forceRun:
+        description: Force the cache to be rebuilt.
+        default: 'false'
+  push:
```
Comment (branberry) [inline .github/workflows/deploy-prd-enhanced-cache.yml:9]: NOTE: Remove this before merging

File: .github/workflows/deploy-prd-enhanced-cache.yml [hunk 2]
```diff
@@ -20,7 +29,9 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
       - name: Rebuild Cache if New Snooty Parser Version
-        uses: mongodb/docs-worker-actions/rebuild-parse-cache@main
+        uses: mongodb/docs-worker-actions/rebuild-parse-cache@cache-update-hotfix
```
Comment (branberry) [inline .github/workflows/deploy-prd-enhanced-cache.yml:32]: NOTE: revert this after merging [this PR](https://github.com/mongodb/docs-worker-actions/pull/80)

File: src/cache-updater/index.ts [hunk 3]
```diff
@@ -7,15 +7,27 @@ import { S3Client } from '@aws-sdk/client-s3';
...
 const readdirAsync = promisify(fs.readdir);
-const SNOOTY_CACHE_BUCKET_NAME = process.env.SNOOTY_CACHE_BUCKET_NAME;
+const { SNOOTY_CACHE_BUCKET_NAME, GITHUB_BOT_USERNAME, GITHUB_BOT_PASSWORD } = process.env;
 
 if (!SNOOTY_CACHE_BUCKET_NAME) throw new Error('ERROR! SNOOTY_CACHE_BUCKET_NAME is not defined');
 
 async function cloneDocsRepo(repoName: string, repoOwner: string) {
+  if (!GITHUB_BOT_USERNAME) {
+    const errorMessage = `ERROR! GITHUB_BOT_USERNAME is not set. ${repoOwner}/${repoName} will not have their cache updated`;
+    console.error(`ERROR! GITHUB_BOT_USERNAME is not set. ${repoOwner}/${repoName} will not have their cache updated`);
```
Comment (rayangler) [inline src/cache-updater/index.ts:17]: (Nit) Should we consolidate to `console.error(errorMessage)`?

**Review by rayangler — APPROVED** [review]
Thanks for the context on the problem. Lgtm with one nit

**Review by mmeigs — APPROVED** [review]

**Pull request merged**